### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by ghost**
Add photo of mentor (Saptak Sengupta)

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://nolik2000.github.io/gci15.fossasia.org

File: _data/mentors.yml [hunk 1]
```diff
@@ -203,4 +203,11 @@
...
   facebook:
   stackexchange: 
-  bio: 
+  bio:
+  
+- name: saptaks
+  github: SaptakS
+  image: saptak.jpg
+  blog: http://blog.fossasia.org/
+  facebook: https://www.facebook.com/s4p74k.s3ngup74
```
Comment (jigyasa-grover) [inline _data/mentors.yml]: No need for the entire URL, `facebook: s4p74k.s3ngup74` shall suffice.

**Comment by jigyasa-grover** [comment]
- Squash Commits. [This](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) might help.
- Give a better description of the PR

Don't open a new PR every time (wrt PR #391 ) @nolik2000  , Try improving the previous instead :)

**Comment by SaptakS** [comment]
Also resolve the merge conflicts.

**Pull request closed without merging**